lastpass-signup.md

```diff
@@ -0,0 +1,63 @@
+# Sign Up for LastPass
+These instructions will help you sign up for LastPass as a free user.  If you already have a LastPass account, you can skip these steps and move on to [viewing a password that has been shared with you](./lastpass-view.md).
+
+## Go to the LastPass Website
+Go to [https://lastpass.com](https://lastpass.com) and click on the "Get LastPass Free" button.
+
+![Go to the LastPass Website](./img/lastpass-signup-1.png)
+
+## Enter Your Information
+Enter your username and password into the signup form.  Your password should be at least 12 characters long, with a number, lowercase letter, and uppercase number.  You should use a password that you can remember, or one that you have recorded in your own personal password manager (if you have one).  The ideal password is one that does not contain common information, such as your name, email, or phone number and is not easily guessed.
+
+### IMPORTANT NOTE!
+Make sure you record this password somewhere safe.  Your vault is secured with this password, and LastPass does not know what the password is.  If you have to reset it, it will erase the contents of your account.  Any passwords created by you will be deleted.  See [protecting your vault](./lastpass-protect.md) for more information.
```

Comment:
```suggestion
Make sure you record this password somewhere safe.  Your vault is secured with this password, and LastPass does not know what the password is.  If you have to reset it, it will erase the contents of your account including your vault.  Any passwords created by you will be deleted.  See [protecting your vault](./lastpass-protect.md) for more information.
```